pbiapi/pbiapi.py

```diff
@@ -104,20 +104,38 @@ def create_workspace(self, name):
             self.force_raise_http_error(response)
 
     @check_token
-    def add_users_to_workspace(self, workspace_name, users):
+    def add_user_to_workspace(self, workspace_name, user):
         self.get_workspaces()
         workspace_id = self.find_workspace_id_by_name(workspace_name, raise_if_missing=True)
 
-        # Workspace exists, lets add users:
+        # Workspace exists, lets add user:
         url = self.base_url + f"groups/{workspace_id}/users"
-        response = requests.post(url, data=users, headers=self.headers)
+        response = requests.post(url, data=user, headers=self.headers)
 
         if response.status_code == HTTP_OK_CODE:
             print(f"Added users to workspace '{workspace_name}'")
         else:
             print(f"Failed to add users to workspace '{workspace_name}':")
             self.force_raise_http_error(response)
 
+    @check_token
+    def get_users_from_workspace(self, name):
+        self.get_workspaces()
+        workspace_id = self.find_workspace_id_by_name(name)
+
+        if workspace_id:
+            url = "https://api.powerbi.com/v1.0/myorg/groups/{groupId}/users".format(groupId=workspace_id)
+
+            response = requests.get(url, headers=self.headers)
+            if response.status_code == 200:
+                return response.json()['value']
+            else:
+                print("Error getting users from workspace")
+                print(response.text)
+                return []
+        else:
```

Comment:
This happens if workspace doesnt exist? Imo returning an empty list tells the caller that the workspace did not have any users - and not that it didnt exist. I vote in favor of raising an exception. I suggest you remove this part (else -> return []) and instead change this call above to:
```python
workspace_id = self.find_workspace_id_by_name(name, raise_if_missing=True)
```